pages/cloudflare/caching.mdx

```diff
@@ -70,3 +70,82 @@ const config: OpenNextConfig = {
 
 export default config;
 ```
+
+
+<Callout>
+  The `direct` mode for the queue is intended for debugging purposes and is not recommended for use in production. We are actively working on a solution that will be suitable for production.
```

Comment:
Question: are the spaces there expected (I see the same l86)

Reply:
Yeah it's how JSX would normally be formatted